app/code/Magento/Newsletter/Model/Subscriber.php

```diff
@@ -353,11 +353,7 @@ public function isStatusChanged()
      */
     public function isSubscribed()
     {
-        if ($this->getId() && $this->getStatus() == self::STATUS_SUBSCRIBED) {
-            return true;
-        }
-
-        return false;
+        return $this->getId() && $this->getStatus() === self::STATUS_SUBSCRIBED;
```

Comment:
Hi @will-in-wi,
Looks like integration & Web API tests started failing after changing `==` to `===`. Looks like sometime `$this->getStatus()` returns string.

I suggest to keep strict comparison there and add converting status to int.
```suggestion
        return $this->getId() && (int)$this->getStatus() === self::STATUS_SUBSCRIBED;
```

Also as @VladimirZaets said - please sign CLA, otherwise we can't accept your Pull Request.

Reply:
Thanks! I'm currently working on getting corporate approval to sign the CLA.